### PR TITLE
created file

### PR DESCRIPTION
**Pull request by Freederth**

### CODE REVIEW

File: Merge.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here] (https://www.markdowntutorial.com) \
```
Comment (praveenscience) [inline Merge.md]: Remove the space.

Comment (praveenscience) [reply Merge.md]: Why there's a `\` at the end of the line?

File: Merge.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+Also, Praveen has been nice to provide us his tutorials and cheatsheets. \
```
Comment (praveenscience) [inline Merge.md]: Why slash?

File: Merge.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+Github commands: [Here] (https://blog.praveen.science/git-shortcuts/) \
```
Comment (praveenscience) [inline Merge.md]: Make all these links into list items by adding `*`.

File: Merge.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+Contributing in a repository tutorial: [Here] (https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md) \
```
Comment (praveenscience) [inline Merge.md]: No need of slash in the last line.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please kindly change these and send me.

File: Merge.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here](https://www.markdowntutorial.com)\
```
Comment (praveenscience) [inline Merge.md]: No need of `\`.

File: Merge.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+Also, Praveen has been nice to provide us his tutorials and cheatsheets.
+
+Github commands: [Here](https://blog.praveen.science/git-shortcuts/)\
```
Comment (praveenscience) [inline Merge.md]: No need of `\`.

**Review by praveenscience — CHANGES_REQUESTED** [review]
No need of `\`. Add `*` to make them as list items.

File: GitHub_stuff.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here](https://www.markdowntutorial.com)\
+Also, Praveen has been nice to provide us his tutorials and cheatsheets.
+
+<li>Github commands: [Here](https://blog.praveen.science/git-shortcuts/)
```
Comment (praveenscience) [inline GitHub_stuff.md]: Check the way how you need to use markdown. I told you to append `*` and a space in front of the lines.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: GitHub_stuff.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here](https://www.markdowntutorial.com)
+Also, Praveen has been nice to provide us his tutorials and cheatsheets.
+
+*Github commands: [Here](https://blog.praveen.science/git-shortcuts/)
+*Contributing in a repository tutorial: [Here](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md)
```
Comment (praveenscience) [inline GitHub_stuff.md]: Missed a space after `*`. Check the preview on how it looks.

File: Merge.md [hunk 9]
```diff
@@ -0,0 +1 @@
+Chirag3
```
Comment (praveenscience) [inline Merge.md]: Delete this file or rename it to `Merge.txt`.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: Merge.md (hunk 9 above)
Comment (Freederth) [reply Merge.md]: done

File: GitHub_stuff.md (hunk 8 above)
Comment (Freederth) [reply GitHub_stuff.md]: ok, done

Comment (praveenscience) [inline GitHub_stuff.md]: Still missing a space after `*`. I was expecting something like this:

```suggestion
* Github commands: [Here](https://blog.praveen.science/git-shortcuts/)
* Contributing in a repository tutorial: [Here](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md)
```

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Can you try to resolve the conflicts and push it? 😁 Everything looks good.

**Comment by Freederth** [comment]
> Can you try to resolve the conflicts and push it? 😁 Everything looks good.

It doesn't let me from here, let's see if i can do it from command line

**Comment by Freederth** [comment]
image

**Comment by praveenscience** [comment]
Closing due to merge conflicts.

**Pull request closed without merging**